### PR TITLE
Feat#29: create my profile page

### PR DESCRIPTION
**Pull request by Minya0606**
| Criticality                    | Impact                       | Application Tested? | Approvals Required |
| ------------------------------ | ---------------------------- | ------------------- | ------------------ |
| ❗️Critical / ⭕️ Non-Critical | 🟩 LOW / 🟨 MEDIUM / 🟥 HIGH | ✅ YES / ❌ NO      | 2                  |

---

### CODE REVIEW

File: src/pages/tenders.tsx [hunk 1]
```diff
@@ -106,11 +106,7 @@ export const getStaticProps: GetStaticProps = async () => {
   }
 };
 
-const MyTendersPage: FC<IProps> = ({
-  tenders,
-  statusCode = null,
-  host = '',
-}) => {
+const Profile: FC<IProps> = ({ tenders, statusCode = null, host = '' }) => {
```
Comment (kenjishiromajp) [inline src/pages/tenders.tsx:109]: Why have you changed this file?

File: src/pages/profile.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+            <Row className="justify-content-between">
+              <Col md="auto">
+                <h1 className="mb-3">My Profile</h1>
+              </Col>
+            </Row>
+            <Row className="justify-content-center">
+              <Col>
+                <Card body style={{ width: '80%' }}>
+                  <Row>
+                    <Col md="3">
```
Comment (kenjishiromajp) [inline src/pages/profile.tsx:93]: `<Col md={3}> ` It expects number as parameter.

File: src/pages/profile.tsx [hunk 3]
```diff
@@ -0,0 +1,182 @@
+import { GetStaticProps } from 'next';
```
Comment (kenjishiromajp) [inline src/pages/profile.tsx:1]: Take a screenshot how the site looks like...

**Review by kenjishiromajp — CHANGES_REQUESTED** [review]

File: src/pages/tenders.tsx (hunk 1 above)
Comment (Minya0606) [reply src/pages/tenders.tsx:109]: I think it was an accident.. I will fix it later

File: src/pages/profile.tsx (hunk 2 above)
Comment (Minya0606) [reply src/pages/profile.tsx:93]: Got it

File: src/pages/profile.tsx (hunk 3 above)
Comment (Minya0606) [reply src/pages/profile.tsx:1]: Okay! Sorry I didn't notice that. I will do it

**Comment by Minya0606** [comment]
## Change the content to show local business profile
* I am not sure which section on the Figma is suit for profile page, so I basically follow the "Local business" section to list profiles. 
* Fix those changed parts 

screenshot

**Pull request closed without merging**